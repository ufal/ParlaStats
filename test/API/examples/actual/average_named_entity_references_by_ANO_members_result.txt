[{'affiliation.person_id': 'RobertStrzinek.1968', 'average_named_entity_references': 15.0}, {'affiliation.person_id': 'VeraJourova.1964', 'average_named_entity_references': 13.152380952380952}, {'affiliation.person_id': 'AndrejBabis.1954', 'average_named_entity_references': 12.828511151295961}, {'affiliation.person_id': 'RichardBrabec.1966', 'average_named_entity_references': 10.931332236842104}, {'affiliation.person_id': 'MartinKolovratnik.1975', 'average_named_entity_references': 10.742249388753056}, {'affiliation.person_id': 'StanislavBerkovec.1956', 'average_named_entity_references': 10.710441334768568}, {'affiliation.person_id': 'MichalRatiborsky.1981', 'average_named_entity_references': 10.47887323943662}, {'affiliation.person_id': 'LubomirMetnar.1967', 'average_named_entity_references': 9.959677419354838}, {'affiliation.person_id': 'MatejFichtner.1984', 'average_named_entity_references': 9.924229808492923}, {'affiliation.person_id': 'AlenaSchillerova.1964', 'average_named_entity_references': 9.827352085354025}, {'affiliation.person_id': 'TatanaMala.1981', 'average_named_entity_references': 9.61111111111111}, {'affiliation.person_id': 'PavelSramek.1965', 'average_named_entity_references': 9.543715846994536}, {'affiliation.person_id': 'KarlaSlechtova.1977', 'average_named_entity_references': 9.35924006908463}, {'affiliation.person_id': 'MilanPour.1965', 'average_named_entity_references': 9.21631205673759}, {'affiliation.person_id': 'JosefHajek.1956', 'average_named_entity_references': 9.17668583376895}, {'affiliation.person_id': 'MartinStropnicky.1956', 'average_named_entity_references': 9.12180889861415}, {'affiliation.person_id': 'RadimHolis.1975', 'average_named_entity_references': 9.0}, {'affiliation.person_id': 'JaroslavBzoch.1983', 'average_named_entity_references': 8.650684931506849}, {'affiliation.person_id': 'AdamVojtech.1986', 'average_named_entity_references': 8.56401028277635}, {'affiliation.person_id': 'MilanHnilicka.1973', 'average_named_entity_references': 8.387543252595156}, {'affiliation.person_id': 'RomanaFischerova.1969', 'average_named_entity_references': 8.125}, {'affiliation.person_id': 'DanTok.1959', 'average_named_entity_references': 8.08}, {'affiliation.person_id': 'KarelHavlicek.1969', 'average_named_entity_references': 8.016114592658909}, {'affiliation.person_id': 'StanislavPfleger.1958', 'average_named_entity_references': 7.777777777777778}, {'affiliation.person_id': 'JiriStrycek.1959', 'average_named_entity_references': 7.610619469026549}, {'affiliation.person_id': 'KristynaZelienkova.1974', 'average_named_entity_references': 7.291463414634146}, {'affiliation.person_id': 'MiloslavJanulik.1962', 'average_named_entity_references': 7.278070836473248}, {'affiliation.person_id': 'RadekZlesak.1983', 'average_named_entity_references': 7.2559852670349905}, {'affiliation.person_id': 'KlaraDostalova.1971', 'average_named_entity_references': 7.145543744889616}, {'affiliation.person_id': 'JosefVozdecky.1945', 'average_named_entity_references': 6.834586466165414}, {'affiliation.person_id': 'JanSchiller.1972', 'average_named_entity_references': 6.791666666666667}, {'affiliation.person_id': 'RomanProchazka.1968', 'average_named_entity_references': 6.77612556794713}, {'affiliation.person_id': 'IvanaDobesova.1960', 'average_named_entity_references': 6.699121027721433}, {'affiliation.person_id': 'JanRehounek.1974', 'average_named_entity_references': 6.662886597938145}, {'affiliation.person_id': 'EvaFialova.1982', 'average_named_entity_references': 6.5710088148873655}, {'affiliation.person_id': 'JosefBelica.1978', 'average_named_entity_references': 6.5064935064935066}, {'affiliation.person_id': 'JanaMrackovaVildumetzova.1973', 'average_named_entity_references': 6.449022055763629}, {'affiliation.person_id': 'MilosBabis.1960', 'average_named_entity_references': 6.307420494699646}, {'affiliation.person_id': 'KamalFarhan.1969', 'average_named_entity_references': 6.259036144578313}, {'affiliation.person_id': 'MargitaBalastikova.1967', 'average_named_entity_references': 6.220731707317073}, {'affiliation.person_id': 'PatrikNacher.1974', 'average_named_entity_references': 6.220195611948189}, {'affiliation.person_id': 'PavelRuzicka.1970', 'average_named_entity_references': 6.142857142857143}, {'affiliation.person_id': 'JiriZlatuska.1957', 'average_named_entity_references': 6.104070305272895}, {'affiliation.person_id': 'JiriMasek.1958', 'average_named_entity_references': 6.102002503128911}, {'affiliation.person_id': 'JaroslavFaltynek.1962', 'average_named_entity_references': 6.073582400910298}, {'affiliation.person_id': 'MiloslavaRutova.1947', 'average_named_entity_references': 6.064285714285714}, {'affiliation.person_id': 'BohuslavChalupa.1960', 'average_named_entity_references': 5.929196623944983}, {'affiliation.person_id': 'LenkaDrazilova.1963', 'average_named_entity_references': 5.90625}, {'affiliation.person_id': 'JanKubik.1978', 'average_named_entity_references': 5.8202416918429005}, {'affiliation.person_id': 'IgorNykl.1963', 'average_named_entity_references': 5.739852398523985}, {'affiliation.person_id': 'PetrVrana.1975', 'average_named_entity_references': 5.72093023255814}, {'affiliation.person_id': 'TomasKohoutek.1983', 'average_named_entity_references': 5.707762557077626}, {'affiliation.person_id': 'IvanaMadlova.1966', 'average_named_entity_references': 5.676470588235294}, {'affiliation.person_id': 'PavelJuricek.1956', 'average_named_entity_references': 5.65023597273204}, {'affiliation.person_id': 'MonikaCervickova.1967', 'average_named_entity_references': 5.578461538461538}, {'affiliation.person_id': 'MartinaBerdychova.1971', 'average_named_entity_references': 5.466145833333333}, {'affiliation.person_id': 'AdamKalous.1979', 'average_named_entity_references': 5.431105047748977}, {'affiliation.person_id': 'MarekNovak.1976', 'average_named_entity_references': 5.417889587700908}, {'affiliation.person_id': 'OndrejBabka.1994', 'average_named_entity_references': 5.318731117824774}, {'affiliation.person_id': 'MonikaOborna.1989', 'average_named_entity_references': 5.063100137174211}, {'affiliation.person_id': 'RadekVondracek.1973', 'average_named_entity_references': 5.004417355435502}, {'affiliation.person_id': 'PavelVolcik.1950', 'average_named_entity_references': 5.0}, {'affiliation.person_id': 'MartinKomarek.1961', 'average_named_entity_references': 4.9511494252873565}, {'affiliation.person_id': 'StanislavFridrich.1956', 'average_named_entity_references': 4.942528735632184}, {'affiliation.person_id': 'JosefKott.1970', 'average_named_entity_references': 4.930196078431372}, {'affiliation.person_id': 'RostislavVyzula.1952', 'average_named_entity_references': 4.927466150870406}, {'affiliation.person_id': 'AndreaBabisova.1972', 'average_named_entity_references': 4.913793103448276}, {'affiliation.person_id': 'ZdenekSoukup.1952', 'average_named_entity_references': 4.887859128822984}, {'affiliation.person_id': 'JiriHolecek.1972', 'average_named_entity_references': 4.851063829787234}, {'affiliation.person_id': 'KarelRais.1949', 'average_named_entity_references': 4.812223626026532}, {'affiliation.person_id': 'BarboraKoranova.1984', 'average_named_entity_references': 4.745566166439291}, {'affiliation.person_id': 'JuliusSpicak.1952', 'average_named_entity_references': 4.678947368421053}, {'affiliation.person_id': 'RobertKralicek.1977', 'average_named_entity_references': 4.619489559164733}, {'affiliation.person_id': 'IvaKalatova.1962', 'average_named_entity_references': 4.611111111111111}, {'affiliation.person_id': 'HelenaValkova.1951', 'average_named_entity_references': 4.5271398747390394}, {'affiliation.person_id': 'EvaMatyasova.1959', 'average_named_entity_references': 4.418803418803419}, {'affiliation.person_id': 'RadkaMaxova.1968', 'average_named_entity_references': 4.413535990823057}, {'affiliation.person_id': 'JanaLorencova.1940', 'average_named_entity_references': 4.229700854700854}, {'affiliation.person_id': 'PavelPustejovsky.1972', 'average_named_entity_references': 4.1361228813559325}, {'affiliation.person_id': 'JanaPastuchova.1965', 'average_named_entity_references': 4.029816513761468}, {'affiliation.person_id': 'JiriHlavaty.1948', 'average_named_entity_references': 4.0}, {'affiliation.person_id': 'ZuzanaOzanova.1970', 'average_named_entity_references': 3.8481619605753865}, {'affiliation.person_id': 'ZuzanaSanova.1959', 'average_named_entity_references': 3.6901960784313723}, {'affiliation.person_id': 'JanSedlacek.1979', 'average_named_entity_references': 3.6666666666666665}, {'affiliation.person_id': 'MartinSedlar.1972', 'average_named_entity_references': 3.6318407960199006}, {'affiliation.person_id': 'AlesJuchelka.1976', 'average_named_entity_references': 3.618313689936537}, {'affiliation.person_id': 'VlastimilVozka.1954', 'average_named_entity_references': 3.5881742738589213}, {'affiliation.person_id': 'IvanPilny.1944', 'average_named_entity_references': 3.560336300692384}, {'affiliation.person_id': 'IvoVondrak.1959', 'average_named_entity_references': 3.533707865168539}, {'affiliation.person_id': 'MilanFeranec.1964', 'average_named_entity_references': 3.3526951672862455}, {'affiliation.person_id': 'JaroslavaPuntova.1957', 'average_named_entity_references': 3.3333333333333335}, {'affiliation.person_id': 'BerenikaPestova.1971', 'average_named_entity_references': 3.324324324324324}, {'affiliation.person_id': 'RomanKubicek.1963', 'average_named_entity_references': 3.275767195767196}, {'affiliation.person_id': 'JanaHanzlikova.1966', 'average_named_entity_references': 3.2058823529411766}, {'affiliation.person_id': 'FrantisekPetrtyl.1959', 'average_named_entity_references': 3.134020618556701}, {'affiliation.person_id': 'JaroslavKytyr.1963', 'average_named_entity_references': 3.0962199312714778}, {'affiliation.person_id': 'VeraAdamkova.1954', 'average_named_entity_references': 3.0535349407437677}, {'affiliation.person_id': 'PavelCihak.1960', 'average_named_entity_references': 2.9375}, {'affiliation.person_id': 'KarelTurecek.1971', 'average_named_entity_references': 2.902439024390244}, {'affiliation.person_id': 'PavelPlzak.1960', 'average_named_entity_references': 2.8396054073803434}, {'affiliation.person_id': 'DavidPrazak.1979', 'average_named_entity_references': 2.8087167070217918}, {'affiliation.person_id': 'DavidKasal.1969', 'average_named_entity_references': 2.6520459228731235}, {'affiliation.person_id': 'JaroslavaPokornaJermanova.1970', 'average_named_entity_references': 2.5718598275919016}, {'affiliation.person_id': 'LadislavOklestek.1961', 'average_named_entity_references': 2.543336439888164}, {'affiliation.person_id': 'VeraProchazkova.1954', 'average_named_entity_references': 2.3538888888888887}, {'affiliation.person_id': 'BronislavSchwarz.1966', 'average_named_entity_references': 2.2301435406698564}, {'affiliation.person_id': 'MarcelaMelkova.1971', 'average_named_entity_references': 2.2052505966587113}, {'affiliation.person_id': 'RobertPelikan.1979', 'average_named_entity_references': 2.142857142857143}, {'affiliation.person_id': 'AndreaBrzobohata.1971', 'average_named_entity_references': 1.95}, {'affiliation.person_id': 'DavidStolpa.1983', 'average_named_entity_references': 1.9268292682926829}, {'affiliation.person_id': 'PetrVenhoda.1986', 'average_named_entity_references': 1.9}, {'affiliation.person_id': 'JanVolny.1959', 'average_named_entity_references': 1.7398293029871976}, {'affiliation.person_id': 'PetrSadovsky.1969', 'average_named_entity_references': 1.699410609037328}, {'affiliation.person_id': 'JanRichter.1967', 'average_named_entity_references': 1.6777251184834123}, {'affiliation.person_id': 'JiriBlaha.1961', 'average_named_entity_references': 1.6445993031358885}, {'affiliation.person_id': 'IvanJac.1956', 'average_named_entity_references': 1.5}, {'affiliation.person_id': 'MilanBrazdil.1962', 'average_named_entity_references': 1.4398950131233597}, {'affiliation.person_id': 'PavelStanek.1967', 'average_named_entity_references': 1.4}, {'affiliation.person_id': 'PremyslMalis.1980', 'average_named_entity_references': 0.9642857142857143}, {'affiliation.person_id': 'RenataOulehlova.1968', 'average_named_entity_references': 0.0}, {'affiliation.person_id': 'IrenaBlazkova.1969', 'average_named_entity_references': 0.0}]

+--------------------------------+---------------------------------+
|     affiliation.person_id      | average_named_entity_references |
+--------------------------------+---------------------------------+
|      RobertStrzinek.1968       |               15.0              |
|        VeraJourova.1964        |        13.152380952380952       |
|        AndrejBabis.1954        |        12.828511151295961       |
|       RichardBrabec.1966       |        10.931332236842104       |
|     MartinKolovratnik.1975     |        10.742249388753056       |
|     StanislavBerkovec.1956     |        10.710441334768568       |
|     MichalRatiborsky.1981      |        10.47887323943662        |
|       LubomirMetnar.1967       |        9.959677419354838        |
|       MatejFichtner.1984       |        9.924229808492923        |
|     AlenaSchillerova.1964      |        9.827352085354025        |
|        TatanaMala.1981         |         9.61111111111111        |
|        PavelSramek.1965        |        9.543715846994536        |
|      KarlaSlechtova.1977       |         9.35924006908463        |
|         MilanPour.1965         |         9.21631205673759        |
|        JosefHajek.1956         |         9.17668583376895        |
|     MartinStropnicky.1956      |         9.12180889861415        |
|        RadimHolis.1975         |               9.0               |
|       JaroslavBzoch.1983       |        8.650684931506849        |
|        AdamVojtech.1986        |         8.56401028277635        |
|       MilanHnilicka.1973       |        8.387543252595156        |
|     RomanaFischerova.1969      |              8.125              |
|          DanTok.1959           |               8.08              |
|       KarelHavlicek.1969       |        8.016114592658909        |
|     StanislavPfleger.1958      |        7.777777777777778        |
|        JiriStrycek.1959        |        7.610619469026549        |
|    KristynaZelienkova.1974     |        7.291463414634146        |
|      MiloslavJanulik.1962      |        7.278070836473248        |
|        RadekZlesak.1983        |        7.2559852670349905       |
|      KlaraDostalova.1971       |        7.145543744889616        |
|       JosefVozdecky.1945       |        6.834586466165414        |
|        JanSchiller.1972        |        6.791666666666667        |
|      RomanProchazka.1968       |         6.77612556794713        |
|       IvanaDobesova.1960       |        6.699121027721433        |
|        JanRehounek.1974        |        6.662886597938145        |
|        EvaFialova.1982         |        6.5710088148873655       |
|        JosefBelica.1978        |        6.5064935064935066       |
| JanaMrackovaVildumetzova.1973  |        6.449022055763629        |
|        MilosBabis.1960         |        6.307420494699646        |
|        KamalFarhan.1969        |        6.259036144578313        |
|    MargitaBalastikova.1967     |        6.220731707317073        |
|       PatrikNacher.1974        |        6.220195611948189        |
|       PavelRuzicka.1970        |        6.142857142857143        |
|       JiriZlatuska.1957        |        6.104070305272895        |
|         JiriMasek.1958         |        6.102002503128911        |
|     JaroslavFaltynek.1962      |        6.073582400910298        |
|      MiloslavaRutova.1947      |        6.064285714285714        |
|      BohuslavChalupa.1960      |        5.929196623944983        |
|      LenkaDrazilova.1963       |             5.90625             |
|         JanKubik.1978          |        5.8202416918429005       |
|         IgorNykl.1963          |        5.739852398523985        |
|         PetrVrana.1975         |         5.72093023255814        |
|       TomasKohoutek.1983       |        5.707762557077626        |
|       IvanaMadlova.1966        |        5.676470588235294        |
|       PavelJuricek.1956        |         5.65023597273204        |
|     MonikaCervickova.1967      |        5.578461538461538        |
|     MartinaBerdychova.1971     |        5.466145833333333        |
|        AdamKalous.1979         |        5.431105047748977        |
|        MarekNovak.1976         |        5.417889587700908        |
|        OndrejBabka.1994        |        5.318731117824774        |
|       MonikaOborna.1989        |        5.063100137174211        |
|      RadekVondracek.1973       |        5.004417355435502        |
|        PavelVolcik.1950        |               5.0               |
|       MartinKomarek.1961       |        4.9511494252873565       |
|     StanislavFridrich.1956     |        4.942528735632184        |
|         JosefKott.1970         |        4.930196078431372        |
|      RostislavVyzula.1952      |        4.927466150870406        |
|      AndreaBabisova.1972       |        4.913793103448276        |
|       ZdenekSoukup.1952        |        4.887859128822984        |
|        JiriHolecek.1972        |        4.851063829787234        |
|         KarelRais.1949         |        4.812223626026532        |
|      BarboraKoranova.1984      |        4.745566166439291        |
|       JuliusSpicak.1952        |        4.678947368421053        |
|      RobertKralicek.1977       |        4.619489559164733        |
|        IvaKalatova.1962        |        4.611111111111111        |
|       HelenaValkova.1951       |        4.5271398747390394       |
|       EvaMatyasova.1959        |        4.418803418803419        |
|        RadkaMaxova.1968        |        4.413535990823057        |
|       JanaLorencova.1940       |        4.229700854700854        |
|     PavelPustejovsky.1972      |        4.1361228813559325       |
|      JanaPastuchova.1965       |        4.029816513761468        |
|        JiriHlavaty.1948        |               4.0               |
|       ZuzanaOzanova.1970       |        3.8481619605753865       |
|       ZuzanaSanova.1959        |        3.6901960784313723       |
|        JanSedlacek.1979        |        3.6666666666666665       |
|       MartinSedlar.1972        |        3.6318407960199006       |
|       AlesJuchelka.1976        |        3.618313689936537        |
|      VlastimilVozka.1954       |        3.5881742738589213       |
|         IvanPilny.1944         |        3.560336300692384        |
|        IvoVondrak.1959         |        3.533707865168539        |
|       MilanFeranec.1964        |        3.3526951672862455       |
|     JaroslavaPuntova.1957      |        3.3333333333333335       |
|      BerenikaPestova.1971      |        3.324324324324324        |
|       RomanKubicek.1963        |        3.275767195767196        |
|      JanaHanzlikova.1966       |        3.2058823529411766       |
|     FrantisekPetrtyl.1959      |        3.134020618556701        |
|       JaroslavKytyr.1963       |        3.0962199312714778       |
|       VeraAdamkova.1954        |        3.0535349407437677       |
|        PavelCihak.1960         |              2.9375             |
|       KarelTurecek.1971        |        2.902439024390244        |
|        PavelPlzak.1960         |        2.8396054073803434       |
|        DavidPrazak.1979        |        2.8087167070217918       |
|        DavidKasal.1969         |        2.6520459228731235       |
| JaroslavaPokornaJermanova.1970 |        2.5718598275919016       |
|     LadislavOklestek.1961      |        2.543336439888164        |
|      VeraProchazkova.1954      |        2.3538888888888887       |
|     BronislavSchwarz.1966      |        2.2301435406698564       |
|      MarcelaMelkova.1971       |        2.2052505966587113       |
|       RobertPelikan.1979       |        2.142857142857143        |
|     AndreaBrzobohata.1971      |               1.95              |
|        DavidStolpa.1983        |        1.9268292682926829       |
|        PetrVenhoda.1986        |               1.9               |
|         JanVolny.1959          |        1.7398293029871976       |
|       PetrSadovsky.1969        |        1.699410609037328        |
|        JanRichter.1967         |        1.6777251184834123       |
|         JiriBlaha.1961         |        1.6445993031358885       |
|          IvanJac.1956          |               1.5               |
|       MilanBrazdil.1962        |        1.4398950131233597       |
|        PavelStanek.1967        |               1.4               |
|       PremyslMalis.1980        |        0.9642857142857143       |
|      RenataOulehlova.1968      |               0.0               |
|       IrenaBlazkova.1969       |               0.0               |
+--------------------------------+---------------------------------+
