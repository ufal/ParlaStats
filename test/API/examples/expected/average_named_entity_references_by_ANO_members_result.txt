[{'affiliation.person_id': 'AndrejBabis.1954', 'average_named_entity_references': 81.39669421487604}, {'affiliation.person_id': 'VeraJourova.1964', 'average_named_entity_references': 50.0}, {'affiliation.person_id': 'RichardBrabec.1966', 'average_named_entity_references': 36.2}, {'affiliation.person_id': 'JosefKott.1970', 'average_named_entity_references': 22.4390243902439}, {'affiliation.person_id': 'AdamVojtech.1986', 'average_named_entity_references': 21.5}, {'affiliation.person_id': 'KarelHavlicek.1969', 'average_named_entity_references': 18.666666666666668}, {'affiliation.person_id': 'DanTok.1959', 'average_named_entity_references': 16.333333333333332}, {'affiliation.person_id': 'RobertPelikan.1979', 'average_named_entity_references': 16.0}, {'affiliation.person_id': 'JanaMrackovaVildumetzova.1973', 'average_named_entity_references': 12.326923076923077}, {'affiliation.person_id': 'LubomirMetnar.1967', 'average_named_entity_references': 12.0}, {'affiliation.person_id': 'BronislavSchwarz.1966', 'average_named_entity_references': 12.0}, {'affiliation.person_id': 'MarekNovak.1976', 'average_named_entity_references': 12.0}, {'affiliation.person_id': 'KlaraDostalova.1971', 'average_named_entity_references': 11.068965517241379}, {'affiliation.person_id': 'PetrVrana.1975', 'average_named_entity_references': 11.0}, {'affiliation.person_id': 'AlenaSchillerova.1964', 'average_named_entity_references': 10.455696202531646}, {'affiliation.person_id': 'RadekVondracek.1973', 'average_named_entity_references': 10.06561203319502}, {'affiliation.person_id': 'JaroslavFaltynek.1962', 'average_named_entity_references': 7.722614840989399}, {'affiliation.person_id': 'ZdenekSoukup.1952', 'average_named_entity_references': 7.5}, {'affiliation.person_id': 'RomanKubicek.1963', 'average_named_entity_references': 7.4}, {'affiliation.person_id': 'BerenikaPestova.1971', 'average_named_entity_references': 7.333333333333333}, {'affiliation.person_id': 'MiloslavJanulik.1962', 'average_named_entity_references': 7.0}, {'affiliation.person_id': 'IvoVondrak.1959', 'average_named_entity_references': 6.5}, {'affiliation.person_id': 'JiriZlatuska.1957', 'average_named_entity_references': 6.5}, {'affiliation.person_id': 'JosefHajek.1956', 'average_named_entity_references': 6.0}, {'affiliation.person_id': 'JaroslavaPokornaJermanova.1970', 'average_named_entity_references': 5.787781350482315}, {'affiliation.person_id': 'JanaHanzlikova.1966', 'average_named_entity_references': 5.5}, {'affiliation.person_id': 'RobertKralicek.1977', 'average_named_entity_references': 5.0}, {'affiliation.person_id': 'MartinStropnicky.1956', 'average_named_entity_references': 4.1875}, {'affiliation.person_id': 'MartinKolovratnik.1975', 'average_named_entity_references': 3.57847533632287}, {'affiliation.person_id': 'MilanBrazdil.1962', 'average_named_entity_references': 3.3548387096774195}, {'affiliation.person_id': 'RomanaFischerova.1969', 'average_named_entity_references': 3.0}, {'affiliation.person_id': 'JaroslavBzoch.1983', 'average_named_entity_references': 3.0}, {'affiliation.person_id': 'JanVolny.1959', 'average_named_entity_references': 2.1560283687943262}, {'affiliation.person_id': 'JiriBlaha.1961', 'average_named_entity_references': 2.0}, {'affiliation.person_id': 'HelenaValkova.1951', 'average_named_entity_references': 1.3804347826086958}, {'affiliation.person_id': 'RadkaMaxova.1968', 'average_named_entity_references': 1.0}, {'affiliation.person_id': 'MatejFichtner.1984', 'average_named_entity_references': 0.5588235294117647}, {'affiliation.person_id': 'PatrikNacher.1974', 'average_named_entity_references': 0.5}, {'affiliation.person_id': 'OndrejBabka.1994', 'average_named_entity_references': 0.0}, {'affiliation.person_id': 'DavidKasal.1969', 'average_named_entity_references': 0.0}]

+--------------------------------+---------------------------------+
|     affiliation.person_id      | average_named_entity_references |
+--------------------------------+---------------------------------+
|        AndrejBabis.1954        |        81.39669421487604        |
|        VeraJourova.1964        |               50.0              |
|       RichardBrabec.1966       |               36.2              |
|         JosefKott.1970         |         22.4390243902439        |
|        AdamVojtech.1986        |               21.5              |
|       KarelHavlicek.1969       |        18.666666666666668       |
|          DanTok.1959           |        16.333333333333332       |
|       RobertPelikan.1979       |               16.0              |
| JanaMrackovaVildumetzova.1973  |        12.326923076923077       |
|       LubomirMetnar.1967       |               12.0              |
|     BronislavSchwarz.1966      |               12.0              |
|        MarekNovak.1976         |               12.0              |
|      KlaraDostalova.1971       |        11.068965517241379       |
|         PetrVrana.1975         |               11.0              |
|     AlenaSchillerova.1964      |        10.455696202531646       |
|      RadekVondracek.1973       |        10.06561203319502        |
|     JaroslavFaltynek.1962      |        7.722614840989399        |
|       ZdenekSoukup.1952        |               7.5               |
|       RomanKubicek.1963        |               7.4               |
|      BerenikaPestova.1971      |        7.333333333333333        |
|      MiloslavJanulik.1962      |               7.0               |
|        IvoVondrak.1959         |               6.5               |
|       JiriZlatuska.1957        |               6.5               |
|        JosefHajek.1956         |               6.0               |
| JaroslavaPokornaJermanova.1970 |        5.787781350482315        |
|      JanaHanzlikova.1966       |               5.5               |
|      RobertKralicek.1977       |               5.0               |
|     MartinStropnicky.1956      |              4.1875             |
|     MartinKolovratnik.1975     |         3.57847533632287        |
|       MilanBrazdil.1962        |        3.3548387096774195       |
|     RomanaFischerova.1969      |               3.0               |
|       JaroslavBzoch.1983       |               3.0               |
|         JanVolny.1959          |        2.1560283687943262       |
|         JiriBlaha.1961         |               2.0               |
|       HelenaValkova.1951       |        1.3804347826086958       |
|        RadkaMaxova.1968        |               1.0               |
|       MatejFichtner.1984       |        0.5588235294117647       |
|       PatrikNacher.1974        |               0.5               |
|        OndrejBabka.1994        |               0.0               |
|        DavidKasal.1969         |               0.0               |
+--------------------------------+---------------------------------+
